melosys-eessi-app/src/main/java/no/nav/melosys/eessi/service/journalpostkobling/JournalpostSedKoblingService.java

```diff
@@ -72,6 +72,8 @@ public JournalpostSedKoblingService(
 
     private MelosysEessiMelding opprettEessiMelding(JournalpostSedKobling journalpostSedKobling)
             throws IntegrationException {
+        BUC buc = euxService.hentBuc(journalpostSedKobling.getRinaSaksnummer());
+        final var organisation = buc.hentDokument(journalpostSedKobling.getSedId()).getCreator().getOrganisation();
```

Comment:
Sorry at jeg ikke kom på dette første gang jeg tittet over - men det vil oppstå et problem med landkoder her, da det er hovedsakelig den vi er interessert i. Det er ISO2 vi mottar fra RINA, men en annen type ISO2 enn det vi bruker i NAV.. Dvs. at vi får en annen landkode for Storbritannia og Hellas (UK og EL istedenfor GB og GR). 

Dette kan løses med `LandkodeMapper.mapTilNavLandkode(...)`. Det er kanskje fornuftig å ha avsender som et objekt som inneholder både institusjonsID og landkode?

Reply:
Ja, det virker nå best å ha et objekt for avsender.